.github/workflows/main.yml

```diff
@@ -121,14 +148,31 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
+      - uses: extractions/setup-just@aa5d15c144db4585980a44ebfdd2cf337c4f14cb  # v1.4.0
 
-      - name: Deploy to dokku
-        uses: opensafely-core/dokku-deploy-github-action@v1
-        with:
-          app-name: "job-server"
-          dokku-host: ${{ secrets.DOKKU_HOST }}
-          ssh-private-key: ${{ secrets.DOKKU2_DEPLOY_SSH_KEY }}
-          remote-branch: "main"
+      - name: Build docker image
+        run: |
+          # docker-test step will build the dev image by default, so build the prod image
+          just docker-build prod
```

Comment:
We need to test the prod image in some way here. It's currently broken, but the tests don't catch that.

We should probably have a just docker-smoketest, which builds and runs prod service, and makes sure we can hit it and get 200 response?